__tests__/app.test.js

```diff
@@ -56,6 +57,7 @@ describe("GET/api/articles/:article_id", () => {
         expect(article).toHaveProperty("created_at", expect.any(String));
         expect(article).toHaveProperty("votes", expect.any(Number));
         expect(article).toHaveProperty("article_img_url", expect.any(String));
+        expect(article).toHaveProperty("comment_count", expect.any(String))
```

Comment:
This should be a number! You've not typecasted the result of the count in your sql query into a number, which is why a String is dropping out.

And btw - it would be a stronger test to look for an exact value here